javascript/1768-merge-strings-alternately.js

```diff
@@ -0,0 +1,10 @@
+var mergeAlternately = function (word1, word2) {
+    let res = '';
```

Comment:
Suggestion: avoid string concatenation, since it is immutable primitive data type. String Concatenation will introduce a performance degradation due to copying a creating new memory allocation every time you attempted to mutate the variable. 

Use a string buffer instead

```js
const buffer = [];

for (...) buffer.push(char);

return buffer.join('');
```

Reply:
Thanks, updated the code